src/Momento.Sdk/Config/PrebuiltConfigurations.cs

```diff
@@ -0,0 +1,101 @@
+using System.Collections.Generic;
+using Momento.Sdk.Config.Middleware;
+using Momento.Sdk.Config.Retry;
+using Momento.Sdk.Config.Transport;
+
+namespace Momento.Sdk.Config;
+
+public class PrebuiltConfigurations
```

Comment:
Interested in your thoughts on this name.  I kind of waffled between this name and the more succinct `Configurations`.

Reply:
I'm taking your lead on the names :-). Regarding this and `BaseConfiguration` above, I only wonder if having `IConfiguration`, `Configuration`, and `Configurations` will feel crowded/confusing

Reply:
Fixed in https://github.com/momentohq/client-sdk-dotnet/pull/170/commits/36d3c41e41b57822047a411aeb3f4e2311a4c51d

Reply:
fair question, maybe we try to solicit input on that during bug bash, and just go with what we think is most straightforward for now